### PR TITLE
Implement ApplyConstants for LeOperator and GeOperator

### PR DESCRIPTION
**Pull request by ptomin**
- Implement ApplyConstants for LeOperator and GeOperator
- Add unit tests to verify it

### CODE REVIEW

File: src/UnitTests/Core/OperatorTests.cs [hunk 1]
```diff
@@ -198,9 +198,81 @@ public void AddressOfCreate()
...
+            RunApply(4, 5, new GeOperator(), 0);
+        }
+
+        [Test]
+        public void GeApplyConstants3()
+        {
+            RunApply(6, 6, new GeOperator(), 1);
+        }
+
+
```
Comment (uxmal) [inline src/UnitTests/Core/OperatorTests.cs]: Need unit tests to handle Constant.Invalid as well

Comment (ptomin) [inline src/UnitTests/Core/OperatorTests.cs]: What do you mean?

Comment (uxmal) [inline src/UnitTests/Core/OperatorTests.cs]: Never mind, I'll add those after I take the PR

Comment (ptomin) [inline src/UnitTests/Core/OperatorTests.cs]: .. you implementation of LtOperator and GtOperator does not handle Constant.Invalid. Is it correct?

**Comment by uxmal** [comment]
They all need to handle it. I'm adding support for it right now.

**Pull request merged**